gravitee-apim-console-webui/src/management/api/api-ng-navigation/api-ng-navigation.component.ts

```diff
@@ -56,16 +74,172 @@ export class ApiNgNavigationComponent implements OnInit {
+                  data: {
+                    apiId: api.id,
+                  },
+                  role: 'alertdialog',
+                  id: 'gioApiReviewDialog',
+                  width: GIO_DIALOG_WIDTH.MEDIUM,
+                })
+                .afterClosed()
+                .pipe(takeUntil(this.unsubscribe$))
+                .subscribe();
+            },
+          },
+        });
+      }
+
+      // Only for not API reviewer
+      if (!isApiReviewer && this.constants.env.settings?.apiReview?.enabled && api.workflowState === 'IN_REVIEW') {
+        // not 'api-reviews-u'
+        banners.push({
+          title: 'The API reviewer has been asked to review the changes.',
+          type: 'info',
+        });
+      }
+      if (!isApiReviewer && this.constants.env.settings?.apiReview?.enabled && api.workflowState === 'REVIEW_OK') {
+        // not 'api-reviews-u'
+        banners.push({
+          title: 'The API reviewer has accepted the changes made on this API.',
+          type: 'success',
+        });
+      }
+      if (!isApiReviewer && this.constants.env.settings?.apiReview?.enabled && api.workflowState === 'REQUEST_FOR_CHANGES') {
+        // not 'api-reviews-u'
+        banners.push({
+          title: 'The API reviewer has asked for changes to be made on this API.',
+          type: 'warning',
+        });
+      }
+
+      // Other banners
+      if (this.constants.env.settings?.apiReview?.enabled && api.workflowState === 'DRAFT') {
+        banners.push({
+          title: 'This API is a draft.',
+          type: 'info',
+        });
+      }
+
+      const apiReviewIsOKOrNotNeeded = (this.constants.env.settings?.apiReview?.enabled && api.workflowState === 'REVIEW_OK') || true;
+      const canUpdateApiDefinition = this.permissionService.hasAnyMatching(['api-definition-u']);
+      if (api.deploymentState === 'NEED_REDEPLOY' && apiReviewIsOKOrNotNeeded) {
+        banners.push(
+          canUpdateApiDefinition
+            ? {
+                title: 'This API is out of sync.',
+                type: 'warning',
+                action: {
+                  btnText: 'Deploy API',
+                  onClick: () => {
+                    this.matDialog
+                      .open<ApiConfirmDeploymentDialogComponent, ApiConfirmDeploymentDialogData, ApiConfirmDeploymentDialogResult>(
+                        ApiConfirmDeploymentDialogComponent,
+                        {
+                          data: {
+                            apiId: api.id,
+                          },
+                          role: 'alertdialog',
+                          id: 'gioApiConfirmDeploymentDialog',
+                          width: GIO_DIALOG_WIDTH.MEDIUM,
+                        },
+                      )
+                      .afterClosed()
+                      .pipe(takeUntil(this.unsubscribe$))
+                      .subscribe();
```

Comment:
Shouldn't we update the `lastApiFetch` from the apiV2Service? Because the banner 'out of sync' will not disappear otherwise